templates/android-dot-com/index.html

```diff
@@ -86,7 +86,7 @@
         </div>
       </div>
 
-      <div class="android-drawer mdl-layout__drawer">
+      <div class="android-drawer mdl-layout__drawer" aria-hidden="true">
```

Comment:
So, in this case let's go ahead and move this into JS during init so it works  the same for  *all* existing sites. Free a11y upgrade without any markup changes in templates is a big win.